test/classes/initializers/secondary/inDiffModule2.chpl

```diff
@@ -20,9 +20,10 @@ module B {
   proc main() {
     var f = new Foo(10);
     // Should use the secondary initializer defined in this module
-    // Additionally, we expect the default constructor/initializer to not be
-    // accessible from this scope. (Do we?)
     writeln(f); // expect 10, true
-    delete f;
+
+    // Should use the default initializer generated by the compiler
+    var f2 = new Foo(2, true);
```

Comment:
Did we make an explicit decision that the default initializer should be generated in this case? If so, is it documented in the spec?

Reply:
I believe it has to be generated in this case.  There's no guarantee that a user of the type will always use or import the module that defines the explicit initializer, so the type would be incapable of being initialized from that context without it.